docs/theme/da_theme_skeleton/layout.html

```diff
@@ -4,7 +4,7 @@
 {# TEMPLATE VAR SETTINGS #}
 {%- set url_root = pathto('', 1) %}
 {%- if url_root == '#' %}{% set url_root = '' %}{% endif %}
-{%- set is_index = theme_index_page_boxes and url_root == './' %}
```

Comment:
Why does `url_root` end up being set to `./` for this?

Reply:
`url_root` is the relative path to `/` from the point of view of Sphinx.
Before the canton/daml docs merge, the index page was the only html file in the root so the check `url_root == './'` worked. We now have other pages in `/`, like `/building-applications.html`. For them the relative path to `/` is also `./`.

Reply:
oh I see, I thought it induded the file name so `/a` and `/` are different.